src/systemtask/SystemTask.cpp

```diff
@@ -148,6 +148,13 @@ void SystemTask::Work() {
 
   nrfx_gpiote_in_init(pinTouchIrq, &pinConfig, nrfx_gpiote_evt_handler);
 
+  nrf_gpio_cfg_sense_input(pinPowerPresentIrq, (nrf_gpio_pin_pull_t) NRF_GPIO_PIN_NOPULL, (nrf_gpio_pin_sense_t) GPIO_PIN_CNF_SENSE_Low);
+
+  pinConfig.sense = NRF_GPIOTE_POLARITY_TOGGLE, pinConfig.pull = NRF_GPIO_PIN_NOPULL, pinConfig.is_watcher = false,
```

Comment:
It's probably a  matter of habit, but I'm not a fan of this kind of one-liner. Would you mind changing to one assignment per line?

Reply:
I think this was clang-tidy, updated